sdk/file_sdk/Common/tag.cpp

```diff
@@ -132,6 +112,45 @@ Tag::Tag(
   }
 }
 
+Tag::Tag(const Tag& tag)
+{
+  mLabelId = tag.GetLabelId();
+  mLabelName = tag.GetLabelName();
+  mLabelParentId = tag.GetLabelParentId();
+  mEnabled = tag.GetEnabled();
+  mMethod = tag.GetMethod();

```

Comment:
>mMethod = tag.GetMethod(); [](start = 2, length = 26)

method should be exposed via the extended properties, as we do in AIP.

Open a task for this.